internal/server/config.go

```diff
@@ -9,10 +9,24 @@ import (
 	"github.com/odpf/salt/config"
 )
 
+type JobType string
+
+const (
+	FetchResources             JobType = "fetch_resources"
+	ExpiringAccessNotification JobType = "appeal_expiration_reminder"
+	RevokeExpiredAccess        JobType = "appeal_expiration_revocation"
+)
+
+type JobConfig struct {
+	JobType  JobType
+	Enabled  bool   `mapstructure:"enabled" default:"true"`
+	Interval string `mapstructure:"interval" default:"0 9 * * *"`
```

Comment:
instead of setting `0 9 * * *` as the default value for every `JobConfig` can we keep the default values for each job interval manually?

Reply:
yeah. I wanted to do that to keep backward compatibility, but it meant  - I would have to model these as anonymous structs, and not as a single JobConfig struct if I need that default interval in tags. 

We can add that custom default intervals in the function - `func fetchJobsToRun(config *Config) []JobConfig { ... }`

Reply:
@rahmatrhd @bsushmith what is our final decision on this?

Reply:
@bsushmith what do you think if we add a few lines of code for assigning the default values for these three jobs?